org.aliflang.lang.metainfo.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+	<id>org.aliflang.lang</id>
+	<content_rating type="oars-1.0" />
+	<project_license>GPL-3.0 and LGPL-3.0</project_license>
+	<metadata_license>CC-BY-SA-3.0</metadata_license>
+	<name>Alif Programming Language</name>
+	<summary>لغة ألف, ألف ستوديو, لغة البرمجة العربية, بيئة تطوير, تعليم</summary>
```

Comment:
I would write the summary & description in english & translate them in arabic. The metainfo file can be translated either automatically using gettext at build stage or you can add the translations manually to the file yourself.

Reply:
Done